.evergreen/config.yml

```diff
@@ -532,110 +410,58 @@ functions:
         args:
           - ${PROJECT_DIRECTORY}/.evergreen/run-mongodb-oidc-remote-test.sh
 
-  "run aws auth test with aws credentials as environment variables":
-    - command: shell.exec
-      type: test
-      params:
-        working_dir: "src"
-        shell: bash
-        script: |
-          . .evergreen/scripts/env.sh
-          .evergreen/run-mongodb-aws-test.sh env-creds
-
-  "run aws auth test with aws credentials and session token as environment variables":
-    - command: shell.exec
-      type: test
-      params:
-        working_dir: "src"
-        shell: bash
-        script: |
-          . .evergreen/scripts/env.sh
-          .evergreen/run-mongodb-aws-test.sh session-creds
-
   "run aws ECS auth test":
-    - command: shell.exec
+    - command: subprocess.exec
       type: test
       params:
-        shell: "bash"
+        binary: bash
         working_dir: "src"
-        script: |
-          if [ "${skip_ECS_auth_test}" = "true" ]; then
-             echo "This platform does not support the ECS auth test, skipping..."
-             exit 0
-          fi
-          . .evergreen/scripts/env.sh
-          set -ex
-          cd ${DRIVERS_TOOLS}/.evergreen/auth_aws
-          . ./activate-authawsvenv.sh
-          . aws_setup.sh ecs
-          export MONGODB_BINARIES="$MONGODB_BINARIES";
-          export PROJECT_DIRECTORY="${PROJECT_DIRECTORY}";
-          python aws_tester.py ecs
-          cd -
+        args:
+          - .evergreen/scripts/run-aws-ecs-auth-test.sh
 
   "cleanup":
-    - command: shell.exec
+    - command: subprocess.exec
       params:
+        binary: bash
         working_dir: "src"
-        script: |
-          . .evergreen/scripts/env.sh
-          if [ -f $DRIVERS_TOOLS/.evergreen/csfle/secrets-export.sh ]; then
-            . .evergreen/hatch.sh encryption:teardown
-          fi
-          rm -rf ${DRIVERS_TOOLS} || true
-          rm -f ./secrets-export.sh || true
+        args:
+          - .evergreen/scripts/cleanup.sh
 
   "fix absolute paths":
```

Comment:
This, "windows fix",   "make files executable", and "init test-results" can be replaced with a call to https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/setup.sh

Reply:
This change should take place in a separate PR. There are enough changes already for a single PR.

Reply:
I created https://jira.mongodb.org/browse/PYTHON-4965 to track this.